owasp-top10-2017-apps/a7/comment-killer/deployments/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
```

Comment:
Could you please update these commands with the new `Deployments` folder in mind?

When running `make install` and checking for container logs with `docker logs`, I'm receiving this error:
```sh
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /app/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/app/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```